tests/tests_geomstats/test_binomial.py

```diff
@@ -40,3 +39,41 @@ def test_point_to_pdf(self, n_draws, point, n_samples):
             pmf.append(gs.array([binom.pmf(x, n_draws, point[i]) for x in samples]))
         expected = gs.squeeze(gs.stack(pmf, axis=0))
         self.assertAllClose(result, expected)
+
+
+class TestBinomialMetric(RiemannianMetricTestCase, metaclass=Parametrizer):
+    skip_test_log_after_exp = True
+    # skip_test_exp_after_log = True
+    skip_test_parallel_transport_ivp_is_isometry = True
+    skip_test_parallel_transport_bvp_is_isometry = True
+    # skip_test_geodesic_ivp_belongs = True
+    # skip_test_geodesic_bvp_belongs = True
+    skip_test_exp_geodesic_ivp = True
```

Comment:
Do you know why this one isn't working ? Sorry if we have we discussed this already and I forgot !